### PR TITLE
Add Sign method to handle grpc request

### PR DESCRIPTION
**Pull request by lookuptable**

### CODE REVIEW

File: pkg/server/grpc/server.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	"istio.io/auth/pkg/pki"
+	"istio.io/auth/pkg/pki/ca"
+	pb "istio.io/auth/proto"
+)
+
+type server struct {
+	ca ca.CertificateAuthority
+}
+
+func (s *server) Sign(ctx context.Context, request *pb.Request) (*pb.Response, error) {
```
Comment (wattli) [inline pkg/server/grpc/server.go]: This func does actually two things, validation and sign, and you have Sign method in s.ca.Sign as well, which can be confusing. Considering renaming to make it accurate? Like validateAndSign or something?

File: pkg/server/grpc/server.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	// TODO: handle authenticate here
```
Comment (wattli) [inline pkg/server/grpc/server.go]: s/authenticate/authentication

File: pkg/server/grpc/server.go (hunk 1 above)
Comment (lookuptable) [inline pkg/server/grpc/server.go]: The name of the method is defined in protobuf. 

Maybe we can rename it to something generic like `HandleRequest`?

Comment (lookuptable) [inline pkg/server/grpc/server.go]: Talked off line: we'll do `HandleCSR()`

**Pull request merged**